examples/Microsoft.Spark.CSharp.Examples/Sql/MachineLearning/SentimentAnalysis.cs

```diff
@@ -0,0 +1,122 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.ML;
+using Microsoft.ML.Data;
+using Microsoft.Spark.Sql;
+
+namespace Microsoft.Spark.Examples.Sql.MachineLearning
```

Comment:
Should these be in `Examples.Sql.MachineLearning` or just `Examples.MachineLearning` ?  I think the latter, mostly because we are trying to showcase MachineLearning with these examples.

Reply:
Adjusted to `Examples.MachineLearning.Sentiment` to reflect newest folder structure.